Manager/UserManager.php

```diff
@@ -61,7 +82,7 @@ public function getUserById($userId)
      */
     public function hasRole(UserInterface $user, $role)
     {
-        return in_array(strtoupper($role), $user->getRoles(), true);
+        return $this->authorizationChecker->isGranted($role);
```

Comment:
If this is some kind of support - the author of the ticket is user with his question.
There are only 2 types of users able to write new messages in the ticket - the author and support member.
If you want to mark messages from support with "Admin" label - you just check if message.userObject != ticket.userCreatedObject